db-engine/src/main/java/com/splicemachine/db/catalog/types/TypeDescriptorImpl.java

```diff
@@ -456,6 +456,12 @@ public String	toString()
 		return s;
 	}
 
+    public String toSparkString()
+	{
+		String s = getSQLstring();
+		return s;
```

Comment:
Why not `return getSQLstring();`?

Reply:
Removed toSparkString.  Changed to use getSQLstring.